lua/graph.lua

```diff
@@ -823,9 +843,6 @@ function filter_tags_generic(kv)
 
   if kv["bike_backward"] == "true" then
     oneway_bike = oneway[kv["oneway:bicycle"]]
-    if (oneway_bike == "false" and kv["bicycle:backward"] == "yes") then
```

Comment:
Ya I just tested that route and it is set as a oneway for bikes when by the tagging it seems that it should not be a oneway for bikes. I do not have tiles built for this part of the world with my new code so I cannot test it with my code changes but I do believe it will now be two way with my changes